Makefile

```diff
@@ -37,17 +37,11 @@ uninstall:
 	done;
 	rmdir $(DESTDIR)$(BINDIR)
 
-.PHONY: tag
-tag:
-	git describe --exact-match >/dev/null 2>&1 || git tag -s $(shell date +%Y%m%d)
-	git push --tags
-
 .PHONY: release
 release:
-	mkdir -p releases
-	git archive --prefix=${REPO}-${TAG}/ -o releases/${REPO}-${TAG}.tar.gz ${TAG};
-	gpg --detach-sign -o releases/${REPO}-${TAG}.tar.gz.sig releases/${REPO}-${TAG}.tar.gz
-	hub release create -m "Release: ${TAG}" -a releases/${REPO}-${TAG}.tar.gz.sig -a releases/${REPO}-${TAG}.tar.gz ${TAG}
+	git describe --exact-match >/dev/null 2>&1 || git tag -s $(shell date +%Y%m%d)
```

Comment:
We should most likely error and exit early if `git describe --exact-match` fails, as pushing and executing `gh` does not make sense in this case 🤔

Reply:
Sounds like a sane behavior indeed.

Fixed https://github.com/archlinux/contrib/pull/96/commits/26eccf94fa98184207ddd471eddf68db979056cd